apps/fluent-tester/src/TestComponents/ColorTokens/ColorTokenTest.tsx

```diff
@@ -100,7 +158,17 @@ const themeSections: TestSection[] = [
   {
     name: 'Alias Color Tokens',
     testID: COLORTOKEN_TESTPAGE,
-    component: () => <AliasTokensSwatchList />,
+    component: () => <AliasColorTokensSwatchList />,
+  },
+  {
+    name: 'Global Color Tokens - Neutral Colors',
+    testID: COLORTOKEN_TESTPAGE,
```

Comment:
These testID's are unnecessay. We only need one set to the first test section (in this case 'Alias Color Tokens', the two new ones don't need it